### ISSUE TITLE
Regarding the Recall@k index presented in the paper,

### ISSUE THREAD

**Issue by gyuseok0917** [kaist-ami/Sound2Scene#8]
Hello. I am writing to ask a question about the evaluation with the Recall@k metric in your Sound2Scene paper. I understand that the Sound Encoder was trained through contrastive learning and that the loss function used was InfoNCE Loss, which means that it was trained in the same way as OpenAI's CLIP. If it is the same method as CLIP, then sounds are matched 1:1 with specific images, and even if they are images of the same class, they are processed as negative samples. Therefore, when calculating Recall@k, I understand that the specific image mentioned above must be included in the predicted k images, and if there is no specific image and only images of the same class, it is recognized as false. I am asking whether the measurement was taken into account.
